### ISSUE TITLE
Service Location

### ISSUE THREAD

**Issue by JGezels** [MicrosoftDocs/OfficeDocs-o365seccomp#37]
Can you specify where the attachements are stored during scan phase? (sandbox location) is this in the Tenant region or in the US?

---
#### Documentdetails

⚠ *Dit gedeelte niet bewerken. Het is vereist om problemen te koppelen tussen docs.microsoft.com en GitHub.*

* ID: 16a4e2d7-73c3-464a-75a6-bb3bc54442a7
* Version Independent ID: 629cc24e-c828-41f8-dc47-89ce5dc2e7fa
* Content: [Dynamic delivery and previewing with Office 365 ATP Safe Attachments](https://docs.microsoft.com/nl-be/office365/securitycompliance/dynamic-delivery-and-previewing?redirectSourcePath=%252fen-us%252farticle%252fdynamic-delivery-and-previewing-with-office-365-atp-safe-attachments-f16c9928-8e3d-4219-b994-271dc9a16272)
* Content Source: [SecurityCompliance/dynamic-delivery-and-previewing.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/dynamic-delivery-and-previewing.md)
* Service: **o365-administration**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

**Comment by denisebmsft** [comment]
@JGezels thank you for the feedback. I will look into this.

**Comment by AndreaBarr** [comment]
@JGezels Thank you for this feedback

I am going to get this over the the o365seccomp team to get these locations updated. Thank you for taking the time to make the docs better. 

I was able to get this into the backlog for the O365seccomp team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
